lib/jsanalyze.js

```diff
@@ -171,7 +173,34 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 	// transpile
 	if (opts.transpile) {
 		options.plugins.push(require.resolve('./global-this'));
-		options.presets.push([ env, { targets: opts.targets } ]);
+		options.presets.push([ env, { targets: opts.targets, useBuiltIns: 'entry' } ]);
```

Comment:
According to the [docs](https://babeljs.io/docs/plugins/preset-env/#usebuiltins) useBuiltIns doesn't take the entry/usage args in 6.X, only in 7.X. It's a boolean in 6.X